packages/module/patternfly-docs/content/examples/basic.md

```diff
@@ -4,26 +4,28 @@
 section: extensions
 # Sidenav secondary level section
 # should be the same for all markdown files
-id: AI-infra-ui-components
+id: DeleteModal
 # Tab (react | react-demos | html | html-demos | design-guidelines | accessibility)
 source: react
 # If you use typescript, the name of the interface to display props for
 # These are found through the sourceProps function provided in patternfly-docs.source.js
-propComponents: ['ExtendedButton']
+propComponents: ['DeleteModal']
 ---
 
-import { ExtendedButton } from "@patternfly/ai-infra-ui-components";
+import { DeleteModal } from "@patternfly/ai-infra-ui-components";
+
+Note: this component is an upgrade of an [existing DeleteModal](https://github.com/opendatahub-io/odh-dashboard/blob/main/frontend/src/pages/projects/components/DeleteModal.tsx) in odh-dashboard
 
 ## Basic usage
 
 ### Example
 
-```js file="./Basic.tsx"
+```js file="./DeleteModalBasic.tsx"
 
 ```
 
 ### Fullscreen example
 
-```js file="./Basic.tsx" isFullscreen
+```js file="./DeleteModalBasic.tsx" isFullscreen
```

Comment:
I don't think need a full screen example in this case. 
Some components only make sense in a full screen context. This is probably not one.